Composer/packages/adaptive-flow/src/adaptive-flow-editor/utils/cursorTracker/locateElement.ts

```diff
@@ -19,7 +19,7 @@ export function locateNearestElement(
   filterAttrs?: string[]
 ): SelectorElement {
   // Get elements that meet the filter criteria
-  let elementArr = elements.filter((element) => !filterAttrs || filterAttrs.find((key) => !!element[key]));
+  let elementArr = elements.filter((element) => !filterAttrs || filterAttrs?.find((key) => !!element[key]));
```

Comment:
Given the !filterAttrs || is still there, does this need the .?

Reply:
this is not my code, it's autofix.